composer.json

```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": "^5.3 | ^7.0",
         "psr/container": "^1.0",
-        "dhii/expression-interface": "^0.1"
+        "dhii/expression-interface": "^0.2"
```

Comment:
Since we are accepting expressions, and not extending them, would it make sense to `require-dev` and `suggest` them instead?